### PR TITLE
feat: change to v2 analyze api in home-dashboard widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA!!!
스크린샷 2023-09-15 오후 6 06 37

### Things to Talk About
I'll add refactoring commits for Vue3, soon

### CODE REVIEW

File: apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue [hunk 1]
```diff
@@ -471,7 +492,7 @@ export default {
             state.loading = true;
             await Promise.all([
                 getCount(),
-                getBillingCount(),
+                allReferenceStore.load('costDataSource'),
```
Comment (WANZARGEN) [inline apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue:495]: question: it doesn't work without this line?

**Review by WANZARGEN — APPROVED** [review]
LGTM! I left a question~

File: apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue:495]: Oh it works. I'll delete it.

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Pull request merged**